test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -54,82 +89,6 @@ func TestSchedule100Node3KPods(t *testing.T) {
 	}
 }
 
-// TestSchedule100Node3KNodeAffinityPods schedules 3k pods using Node affinity on 100 nodes.
-func TestSchedule100Node3KNodeAffinityPods(t *testing.T) {
```

Comment:
This can be simulated by extending the datastructure, so there is need to have a separate test function. For example, when we have affinity set in schedulerPerfConfig struct(https://github.com/kubernetes/kubernetes/pull/46117/files/4ba70af63f9793649e3cf5108d6ff5cd74858c9b#diff-3126019cd381f25186f4fa8c8da06ec6R267), this will be automatically run.